packages/stable/indent-runtime-aws-lambda/src/index.ts

```diff
@@ -10,6 +12,10 @@ export function getLambdaHandler({
     event
   ): Promise<APIGatewayProxyResult> => {
     const { headers, multiValueHeaders, body } = event
+
+    await loadSecrets(integrations) // TODO: not here
```

Comment:
by running loadSecrets() here, we ensure that secrets are gathered by checking at the beginning of every request. I know we're limited by the entrypoints that Lambda runs; is there a better way of running an `await` init setup block? We either need to block here if secrets are still loading, or not accept requests until secrets are loaded.